docs/eventing/experimental-features.md

```diff
@@ -0,0 +1,54 @@
+# Experimental features
+
+In Knative we want to keep the innovation alive, experimenting and delivering new features without affecting the stability of the project.
+
+In order to achieve that goal in Knative Eventing, we have a process to include new features.
+This allows users like you to try out new features and provide feedback back to the project.
+
+This document explains how to enable experimental features and which ones are available today.
+
+For more details about the process, the feature phases, quality requirements and guarantees, check out the [Experimental features process documentation](https://github.com/knative/eventing/blob/main/docs/experimental-features.md).
+
+## Before you begin
+
+You must have a Knative cluster running with the Eventing component installed. [Learn more](../install/)
+
+## Experimental features configuration
+
+When installing Eventing, the `config-features` ConfigMap is added to your cluster in the `knative-eventing` namespace.
+In order to enable a feature, you just need to add it to the config map and set its value to `"enabled"`.
```

Comment:
Note: @eric-sap related to the other PR you're doing, it looks like it's both `` and "" 🙂 so `"enabled"`

Reply:
In this particular case (judging by the new-cool-feature: example text below this sentence) it looks like the quotes are literally supposed to be part of the configmap (as opposed to, say, the metadata.name or namespace, which do not have quotes).  I'm unsure as to whether the yaml parser cares one way or the other.  I only mention it here because in the config-kafka configmap, there are no fields that require literal double-quotes around them (nor are there examples of such in the document).

Reply:
That's my bad, i was using true and false as values before, which needs escapes. https://github.com/knative/docs/pull/3714